src/components/30-organisms/footer/index.js

```diff
@@ -20,6 +20,37 @@ const _setMaxHeightToZero = panel => {
   panel.style.maxHeight = '0px';
 };
 
+// hence there is no re-render to set the caret-open-class
+// it has to be done by manually adding the class to the
+// element via javascript
+const _changeStateOfCaret = (caret, isActive) => {
```

Comment:
Please explain once, what a caret is: The caret is a little UI icon visually indicating accordion open/closed state.